tests/phpunit/elementor/core/experiments/test-manager.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+namespace Elementor\Tests\Phpunit\Elementor\Core\Experiments;
+
+use Elementor\Core\experiments\Manager as Experiments_Manager;
+use Elementor\Testing\Elementor_Test_Base;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+class Test_Manager extends Elementor_Test_Base {
+
+	public function tearDown() {
+		parent::tearDown();
+
+		$experiments = $this->elementor()->experiments;
+
+		$features = $experiments->get_features();
+
+		foreach( $features as $feature_name => $feature ) {
+			$experiments->remove_feature( $feature_name );
+		}
+	}
+
+	public function test_add_feature() {
+		$experiments = $this->elementor()->experiments;
+
+		$test_feature_data = [
+			'name' => 'test_feature',
+			'unaccepted_value' => 'bla bla',
+		];
+
+		$test_set = [
+			'description' => '',
+			'status' => Experiments_Manager::STATUS_ALPHA,
+			'default' => Experiments_Manager::STATE_INACTIVE,
+			'name' => 'test_feature',
+			'state' => Experiments_Manager::STATE_DEFAULT,
+		];
+
+		$new_feature = $experiments->add_feature( $test_feature_data );
+
+		$again_added_feature = $experiments->add_feature( $test_feature_data );
+
+		$this->assertEqualSets( $test_set, $new_feature );
+
+		$this->assertEquals( null, $again_added_feature );
+	}
+
+	public function test_get_features() {
+		$experiments = $this->elementor()->experiments;
+
+		// Ensures `get_features` is called correctly even before adding any feature
+		$experiments->get_features();
+
+		$this->add_test_feature();
+
+		$features = $experiments->get_features();
+
+		$test_feature = $experiments->get_features( 'test_feature' );
+
+		$this->assertArrayHaveKeys( [ 'test_feature' ], $features );
+
+		$this->assertNotEmpty( $test_feature );
+	}
+
+	public function test_is_feature_active() {
+		$this->add_test_feature();
+
+		$experiments = $this->elementor()->experiments;
+
+		$is_test_feature_active = $experiments->is_feature_active( 'test_feature' );
+
+		$this->assertFalse( $is_test_feature_active );
+	}
+
+	public function test_is_feature_active__default_activated() {
+		$experiments = $this->elementor()->experiments;
+
+		$default_activated_test_feature_data = [
+			'name' => 'default_activated_test_feature',
+			'default' => Experiments_Manager::STATE_ACTIVE,
+		];
+
+		$experiments->add_feature( $default_activated_test_feature_data );
+
+		$is_default_activated_test_feature_active = $experiments->is_feature_active( 'default_activated_test_feature' );
+
+		$this->assertTrue( $is_default_activated_test_feature_active );
+	}
+
+	public function test_is_feature_active__not_exists() {
+		$experiments = $this->elementor()->experiments;
+
+		$is_non_exist_active = $experiments->is_feature_active( 'not_exists_feature' );
```

Comment:
```suggestion
		$non_exist_active = $experiments->is_feature_active( 'not_exists_feature' );
```

Reply:
Since it's a boolean it should be prefixed with an `is`, just like the method itself.